packages/devtools_app/test/integration_tests/integration.dart

```diff
@@ -198,11 +197,11 @@ class BrowserTabInstance {
     if (message.containsKey('id')) {
       // handle a response: {id: 1}
       final AppResponse response = AppResponse(message);
-      final Completer<AppResponse> completer = _completers.remove(response.id);
+      final Completer<AppResponse>? completer = _completers.remove(response.id);
```

Comment:
null assert this var here instead of doing so twice below

Reply:
done